src/app/app.component.scss

```diff
@@ -29,3 +29,5 @@ button {
   font-size: 200%;
```

Comment:
Do not use style directly on HTML element button is generic selector if you are using generic you should be really mindful as this style will apply to all buttons which is not right instead you could create a class and add to button whenever we need this styling